test/support/api/patients.js

```diff
@@ -31,3 +31,33 @@ Cypress.Commands.add('routePatient', (mutator = _.identity) => {
   })
     .as('routePatient');
 });
+
+Cypress.Commands.add('routePatients', (mutator = _.identity) => {
+  cy
+    .fixture('collections/patients').as('fxPatients')
+    .fixture('collections/groups').as('fxGroups');
+
+  cy.route({
+    url: '/api/patients**', // probably a regex to match for includes or just copy/past it?
+    response() {
+      const data = getResource(_.sample(this.fxPatients, 30), 'patients');
+      const groups = _.sample(this.fxGroups, 2);
+
+      let included = [];
+
+      included = getIncluded(included, groups, 'groups');
+
+      _.each(data, (patient) => {
```

Comment:
only need to wrap the argument if there's more than one for `=>`s    Mind adding `arrow-parens: [2, "as-needed"]` to the eslint.. it _should_ auto fix this if we add that.